tests/py_test.py

```diff
@@ -754,48 +752,62 @@ def make_monster_from_generated_code():
     ''' Use generated code to build the example Monster. '''
 
     b = flatbuffers.Builder(0)
-    string = b.CreateString("MyMonster")
```

Comment:
Shouldn't this test still use the old way of making the object? In effect, it seems you want to add a second, property-based API. It will need to be tested _along with_ the existing API. Do I understand correctly?